crates/biome_js_analyze/tests/specs/nursery/useValidAriaRole/invalid.jsx

```diff
@@ -0,0 +1,6 @@
+<>
```

Comment:
You should add eslint's tests as much as possible
ref: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/__tests__/src/rules/aria-role-test.js